### PR TITLE
Authors

### PR DESCRIPTION
**Pull request by rafaeleyng**
Adds authoring structure.
- Adds the possibility to set the author of a post with: 

``` yaml

---
author: rafael

---
```
- Adds authors register in `authors.yml`.
- Adds authors page, which display all authors.
- Adds individual author page, which display all posts from this author.
- Adds author social links component
- Adds author Gravatar image component.

### CODE REVIEW

File: _includes/author.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% if include.author %}
-  {% assign author = site.authors[include.author] %}
-  <p>{{ author.name }}</p>
+  {% assign author = site.data.authors[include.author] %}
```
Comment (dwayhs) [inline _includes/author.html]: Nice!

I didn't know this jekyll`s feature.

It`s great to load data this way. I think we can start breaking our _config.yml even more.

Comment (rafaeleyng) [inline _includes/author.html]: :+1:

**Comment by dwayhs** [comment]
Great pull request.

For me it`s ready to merge.

Just missed a description. :wink:

**Comment by rafaeleyng** [comment]
@dwayhs I will be adding a couple more commits to this PR, you can merge it now or wait more. If you guys prefer, I can open it only when the stuff is really finished.

**Comment by dwayhs** [comment]
I think it`s nice to open the pull request before the work is complete, but it is important to mark it as WIP (work in progress)

**Comment by rafaeleyng** [comment]
How should I mark it as WIP?

**Comment by dwayhs** [comment]
@rafaeleyng, merge it yourself whenever you think you're comfortable with the feedbacks provided by the team.

**Comment by dwayhs** [comment]
Just marked as wip in the name of the pull request.

We can use tags too. But the name makes it REALLY clear.

**Comment by rafaeleyng** [comment]
@dwayhs Now I see, I will adopt you suggestion of adding WIP to the title.

**Comment by rafaeleyng** [comment]
@dwayhs @marlonbernardes Done.

**Comment by rafaeleyng** [comment]
@dwayhs @marlonbernardes Could you take a look at the pull request? I would like to merge it soon, but it would be nice some revision :+1:

**Comment by marlonbernardes** [comment]
@rafaeleyng I'm gonna review it in the next minutes.

**Comment by marlonbernardes** [comment]
@rafaeleyng Ok to merge!

**Pull request merged**